src/sentry/static/sentry/app/views/organizationIntegrations/integrationListDirectory.tsx

```diff
@@ -201,9 +203,21 @@ export class OrganizationIntegrations extends AsyncComponent<
   }
 
   sortIntegrations(integrations: AppOrProviderOrPlugin[]) {
-    return integrations
-      .sort((a, b) => a.name.localeCompare(b.name))
-      .sort((a, b) => this.getInstallValue(b) - this.getInstallValue(a));
+    const sortByName = (a, b) => a.name.localeCompare(b.name);
+    const sortByPopularity = (a, b) => {
+      const weightA = Number.isInteger(weights[a.slug]) ? weights[a.slug] : 1;
+      const weightB = Number.isInteger(weights[b.slug]) ? weights[b.slug] : 1;
+      return weightA > weightB ? -1 : weightB > weightA ? 1 : 0;
```

Comment:
nit: Would `weightB - weightA` work in your use case?

Reply:
@jeffkwoh yes it does!